src/components/service-metrics/range-picker.njk

```diff
@@ -0,0 +1,73 @@
+{% from "govuk-frontend/govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk-frontend/govuk/components/input/macro.njk" import govukInput %}
+
+<div class="paas-statement-filters">
+  <p class="govuk-body">
+    The graphs below show
+    <strong>{{ period.humanize() }}</strong>
+    points between
+    <strong>{{ rangeStart | nicedatetime }}</strong>
+    and
+    <strong>{{ rangeStop | nicedatetime }}</strong> for the
+    <a href="{{ linkTo(
+      'admin.organizations.spaces.services.view',
+      {organizationGUID: organization.metadata.guid, spaceGUID: space.metadata.guid, serviceGUID: service.metadata.guid}
+      )}}" class="govuk-link">{{ service.entity.name }}</a>
+    service over the last 24 hours.
+  </p>
+
+  <details class="govuk-details" data-module="govuk-details">
+    <summary class="govuk-details__summary">
+      <span class="govuk-details__summary-text">
+        Change time period
+      </span>
+    </summary>
+
+    <ol class="govuk-list">
+      <li>
+        <a href="{{ linkTo('admin.organizations.spaces.services.metrics.redirect', {organizationGUID: organization.metadata.guid, spaceGUID: space.metadata.guid, serviceGUID: service.metadata.guid, offset: '1h'}) }}" class="govuk-list govuk-link--no-visited-state" title="last 1 hour">Last 1 hour</a>
```

Comment:
Should these have `govuk-list` or `govuk-link` 🤔

Reply:
Not certain what you mean. They do, the `govuk-list` is on the `ol` element and the `govuk-link` is on the `a` element.

As per: https://design-system.service.gov.uk/styles/typography/#lists and https://design-system.service.gov.uk/styles/typography/#links

Do you perhaps consider removing these?

Reply:
This one is `<a ... class="govuk-list...>` though, which looks wrong?

Reply:
ah, beautiful!